h2o-r/tests/testdir_algos/stackedensemble/runit_stackedensemble_seed.R

```diff
@@ -0,0 +1,85 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+stackedensemble.metalearner.seed.test <- function() {
+
+    print("Reading in data")
+    train <- h2o.uploadFile(locate("smalldata/testng/higgs_train_5k.csv"),
+    destination_frame = "higgs_train_5k")
+    test <- h2o.uploadFile(locate("smalldata/testng/higgs_test_5k.csv"),
+    destination_frame = "higgs_test_5k")
+    y <- "response"
+    x <- setdiff(names(train), y)
+    train[,y] <- as.factor(train[,y])
+    test[,y] <- as.factor(test[,y])
+    nfolds <- 3  #number of folds for base learners
+
+    print("Training GBM")
+    # Train & Cross-validate a GBM
+    my_gbm <- h2o.gbm(x = x,
+                    y = y,
```

Comment:
(nitpick) spacing is off by two spaces on all the algo functions (GBM, RF and SEs)